src/harness/testcases/WINNF_FT_S_PPR_testcase.py

```diff
@@ -0,0 +1,391 @@
+#    Copyright 2018 SAS Project Authors. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import json
+import os
+import sas
+import sas_testcase
+from sas_test_harness import SasTestHarnessServer, generateCbsdRecords, \
+    generatePpaRecords
+from util import winnforum_testcase, configurable_testcase, writeConfig, \
+  loadConfig, makePpaAndPalRecordsConsistent
+
+
+class PpaProtectionTestcase(sas_testcase.SasTestCase):
+
+  def setUp(self):
+    self._sas, self._sas_admin = sas.GetTestingSas()
+    self._sas_admin.Reset()
+
+  def tearDown(self):
+    pass
+
+  def generate_PPR_1_default_config(self, filename):
+    """ Generates the WinnForum configuration for PPR.1. """
+
+    # Load devices info
+    device_1 = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    # Moving device_1 to a location within 40 meters of PPA zone
+    device_1['installationParam']['latitude'] = 38.8203
+    device_1['installationParam']['longitude'] = -97.2741
+
+    device_2 = json.load(
+      open(os.path.join('testcases', 'testdata', 'device_b.json')))
+    # Moving device_2 to a location outside 40 meters of PPA zone
```

Comment:
meters -> km. (Same in PPR.2)

Reply:
Modified meters to KMs for both testcases